src/Security/OfferSecurityServiceProvider.php

```diff
@@ -96,6 +113,17 @@ function ($app) {
 
                 $security = new MediaSecurity($security);
 
+                /** @var ToggleManager $toggles */
+                $toggles = $app['toggles'];
+                if ($toggles->active('facility-permission', $app['toggles.context'])) {
+                    $security = new SecurityWithUserPermission(
+                        $security,
+                        $app['current_user_identification'],
+                        $app['facility_permission_voter'],
+                        new ClassNameCommandFilter(new StringLiteral(UpdateFacilities::class))
+                    );
+                }
+
```

Comment:
Kunnen we deze bootstrapping nog lichtjes aanpassen zodat we eigenlijk altijd een `SecurityWithUserPermission` met `ClassNameCommandFilter` bootstrappen, zelfs als de feature toggle af staat, maar al dan niet met `UpdateFacilities::class` in de lijst van class names die gefilterd moeten worden (gebaseerd op de feature toggle dan)? Maakt de code iets flexibeler als er later andere commands bij komen die gecontroleerd moeten worden adhv andere feature toggles.

Reply:
Bijvoorbeeld:

    $filterCommands = [];
    
    /** @var ToggleManager $toggles */
    $toggles = $app['toggles'];
    if ($toggles->active('facility-permission', $app['toggles.context'])) {
      $filterCommands[] = new StringLiteral(UpdateFacilities::class);
    }
    
    $security = new SecurityWithUserPermission(
        $security,
        $app['current_user_identification'],
        $app['facility_permission_voter'],
        new ClassNameCommandFilter(...$filterCommands)
    );

(Nog niet getest, we moeten dan even controleren of ClassNameCommandFilter correct om kan gaan met een lege lijst van class names maar ik zou denken van wel.)